vnext/Desktop.UnitTests/MemoryMappedBufferTests.cpp

```diff
@@ -0,0 +1,149 @@
+#include "MemoryMappedBuffer.h"
+#include "utilities.h"
+
+#include <CppUnitTest.h>
+
+#include <shlwapi.h>
+#include <windows.h>
+
+#include <cstring>
+#include <memory>
+
+using facebook::jsi::Buffer;
+using facebook::jsi::JSINativeException;
+using facebook::react::utilities::checkedReinterpretCast;
+using Microsoft::JSI::makeMemoryMappedBuffer;
+using Microsoft::VisualStudio::CppUnitTestFramework::Assert;
+
+namespace {
+
+static constexpr const wchar_t *const testFileName = L"temp.txt";
+
+static void writeTestFile(const char *const content, size_t size) {
+  FILE *testFilePtr;
+
+  // We need pass in the "b" flag to ensure that fwrite modify its input when
+  // outputting to disk.
+  if (_wfopen_s(&testFilePtr, testFileName, L"wb")) {
+    return;
```

Comment:
>return [](start = 4, length = 6)

crash or throw exception

Reply:
Fixed.

---
In reply to: [320914185](https://github.com/microsoft/react-native-windows/pull/3059#discussion_r320914185) [](ancestors = 320914185)